sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1432,6 +1464,20 @@ def task_worker(self):
                         self.port_dict[lport]['cmis_state'] = self.CMIS_STATE_READY
                         continue
 
+                    appl = self.get_cmis_application_desired(api, host_lane_count, host_speed)
+                    if appl < 1:
+                        self.log_error("{}: no suitable app for the port".format(lport))
```

Comment:
@mihirpat1 can you log the appl, lane count, speed?

Reply:
@mihirpat1 is_cmis_application_update_required() still expects application = 1. can you fix?

Reply:
> @mihirpat1 is_cmis_application_update_required() still expects application = 1. can you fix?

@mihirpat1 also the log "Non-default application is not supported"

Reply:
I have addressed this now.